### PR TITLE
Page about staking from the consumer perspective

### PR DESCRIPTION
**Pull request by timothymcmackin**
I intend this page to be part of a new section for Tezos consumers but let's start with a page on staking because we're trying to clarify that topic for all users.

Preview: https://docs-staging-git-staking-site-trili-tech.vercel.app/using/staking

### CODE REVIEW

**Review by beatalipska — APPROVED** [review]
very useful 👏🏻

File: docs/using/staking.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+---
+title: Staking
+authors: "Tim McMackin"
+last_update:
+  date: 17 December 2024
+---
+
+Staking is the process of temporarily locking tez on the Tezos platform.
```
Comment (NicNomadic) [inline docs/using/staking.md:8]: State right from the beginning the usefulness for yourself:
```suggestion
Staking is the process of temporarily locking tez on the Tezos platform in exchange of rewards.
```

File: docs/using/staking.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+Staked tez stays in the staker's account, but they cannot transfer that tez while it is staked.
+They can unstake the tez and make it spendable after a delay.
+Staking is an important part of keeping Tezos secure and earning rewards from it.
```
Comment (NicNomadic) [inline docs/using/staking.md:11]:
```suggestion
Staking is an important part of running the Tezos protocol and keeping the blockchain secure, so this is why stakers earn rewards proportional to the locked funds.
```

File: docs/using/staking.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
+---
+title: Staking
+authors: "Tim McMackin"
+last_update:
+  date: 17 December 2024
+---
+
+Staking is the process of temporarily locking tez on the Tezos platform.
+Staked tez stays in the staker's account, but they cannot transfer that tez while it is staked.
+They can unstake the tez and make it spendable after a delay.
```
Comment (NicNomadic) [inline docs/using/staking.md:10]:
```suggestion
Stakers can unstake the tez at any time, which makes them spendable again after a delay.
```

File: docs/using/staking.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
+---
+title: Staking
+authors: "Tim McMackin"
+last_update:
+  date: 17 December 2024
+---
+
+Staking is the process of temporarily locking tez on the Tezos platform.
+Staked tez stays in the staker's account, but they cannot transfer that tez while it is staked.
```
Comment (NicNomadic) [inline docs/using/staking.md:9]: shouldn't it be plural? "the tez staked" suggests 1 tez, but maybe I'm wrong:
```suggestion
Staked tez stays in the staker's account, but they cannot be spent (e.g., transferred to another account).
```

File: docs/using/staking.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+They can unstake the tez and make it spendable after a delay.
+Staking is an important part of keeping Tezos secure and earning rewards from it.
+
+Two main groups stake on Tezos:
+
+- Bakers must stake 6,000 tez to receive the right to bake blocks and to receive rewards for baking those blocks.
```
Comment (NicNomadic) [inline docs/using/staking.md:15]: Don't assume end users know what is a baker:
```suggestion
- Bakers, the creators of new blocks in Tezos, must stake 6,000 tez to receive the right to "bake" blocks and to receive rewards for baking those blocks.
```

File: docs/using/staking.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+Their staked tez ensures that they bake correctly, because part of their stake is taken ("slashed") if they misbehave.
+For more information about staking for baking purposes, see [Baking](/architecture/bakers).
+
+- Any Tezos user can stake any amount of tez with a baker and earn rewards based on how much they stake.
```
Comment (NicNomadic) [inline docs/using/staking.md:19]: de-duplicate "any amount" and "based on how much", which are detailed just after:
```suggestion
- Any Tezos user can stake tez with a baker and earn rewards for them.
```

**Review by NicNomadic — COMMENTED** [review]
I only reviewed the intro part, until section "Staking as a Tezos user", excluded

File: docs/using/staking.md (hunk 1 above)
Comment (timothymcmackin) [reply docs/using/staking.md:8]: I made this "in exchange for rewards".

File: docs/using/staking.md (hunk 4 above)
Comment (timothymcmackin) [reply docs/using/staking.md]: It's tricky, "tez" can be singular or plural. I made this "but the staker cannot transfer (or spend) that tez."

File: docs/using/staking.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+- Any Tezos user can stake tez with a baker and earn rewards for them.
+In exchange for staking tez with a baker, users automatically receive a portion of the baker's rewards in proportion to how much they stake.
+Users can stake any amount of tez, but there is a limit to how much staked tez a single baker can accept, and bakers must opt in to allow users to stake with them.
+
+Staking has some similarities to the concept of a Smart Rollup bond, but there are important differences; see [Bonds](/architecture/smart-rollups#bonds).
+
+## Staking as a Tezos user
+
+The process of user staking has these main steps:
```
Comment (NicNomadic) [inline docs/using/staking.md:27]: "user staking" sounds weird, it's like you stake the user, 😄 , can we say:
```suggestion
The process of staking has these main steps for a user:
```

File: docs/using/staking.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+1. The user selects a baker as a delegate for their account.
+As described in [Delegating to a baker](/architecture/bakers#delegating-to-a-baker), delegating tez to a baker puts that tez toward the baker's voting and baking power.
+Delegating incurs no risk to the user; the user retains full control of the tez, they can spend it at any time, and they are not punished if the baker misbehaves.
+
+1. The user stakes tez with the baker.
+Staking the tez locks it temporarily, but the tez stays in the user's account.
```
Comment (NicNomadic) [inline docs/using/staking.md:34]: (it's really a question, I just want to learn:)
so ok, "tez" can be singular or plural, but in this case it *is* plural, so don't we have to agree in number?
```suggestion
Staking the tez locks them temporarily, but the tez stay in the user's account.
```

File: docs/using/staking.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+However, if the baker misbehaves, their punishment also affects tez staked with them, so it's important to choose a responsible baker.
+
+1. The user leaves their tez staked for as long as they want.
+During this time the rewards are added to their account automatically.
+
+THe process of unstaking has these main steps:
```
Comment (NicNomadic) [inline docs/using/staking.md:40]:
```suggestion
The process of unstaking has these main steps:
```

File: docs/using/staking.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+
+1. After a delay of about 10 days, the tez are unfrozen in their account and the user can use that tez as usual.
+
+## How to stake
+
+The easiest way to stake is to use the staking web application at https://stake.tezos.com, which walks you through the process.
+
+:::warning
+
+Make sure that you are staking at the right URL (stake.tezos.com).
```
Comment (NicNomadic) [inline docs/using/staking.md:52]: just for security:
```suggestion
Make sure that you are staking at the right URL (https://stake.tezos.com).
```

**Review by NicNomadic — COMMENTED** [review]
LGTM beyond the few remaining threads.

**Pull request merged**